src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.DevOps.tests.ps1

```diff
@@ -32,6 +32,23 @@ InModuleScope Arcus.Scripting.DevOps {
                     $patchResponse.StatusCode | Should -Be 200
                 }
             }
+            #It "Sets the DevOps variable group description with the release name" {
```

Comment:
We can maybe use `-Skip` for this: https://pester-docs.netlify.app/docs/usage/skip

Reply:
Better solution indeed, I changed it to `-Skip` 👍🏻